code/modules/projectiles/projectile/magic.dm

```diff
@@ -678,6 +678,25 @@
 	var/turf/T = get_turf(target)
 	explosion(T, -1, exp_heavy, exp_light, exp_flash, 0, flame_range = exp_fire)
 
+/obj/projectile/magic/aoe/grease
+	name = "blob of grease"
+	icon_state = "grease"
+	damage = 0
+	damage_type = BRUTE
+	nodamage = TRUE
+
+/obj/projectile/magic/aoe/grease/on_hit(target)
+	. = ..()
+	if(ismob(target))
+		var/mob/living/M = target
+		if(M.anti_magic_check())
+			visible_message("<span class='warning'>[src] vanishes into smoke on contact with [target]!</span>")
+			return BULLET_ACT_BLOCK
+	var/turf/T = get_turf(target)
+	for(var/turf/open/OT in range(T, 5))
+		OT.MakeSlippery(TURF_WET_LUBE, 100)
```

Comment:
won't this go through walls?

Reply:
right, need to use view

Reply:
i think view still pierces windows and border-windows, but the solution to that it probably a bit more complex

Reply:
does the original spell require line of effect, or just line of sight? We could turn the ability to go through windows into a feature

Reply:
It's magic, why shouldn't it go through walls

Reply:
that argument always triggers me

Reply:
switched to view